torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3320,7 +3320,10 @@ def large_1d_unique():
     flag = [True, False]
     for dim, descending, stable in product(dims, flag, flag):
         # default schema without stable sort
-        yield SampleInput(small_3d_unique(), dim, descending)
+        if not (dtype == torch.bool and torch.device(device).type == 'cuda'):
+            # bool and cuda requires stable sort for stable results, at least
```

Comment:
adding this because we need stable sort (otherwise it'll fail inductor run which runs on the compiled graph and eager mode and produce different indices). We have the stable sort below and they run fine

Reply:
The samples seem to be coming from `small_3d_unique`. Isn't it stable in this case?

Reply:
small_3d_unique only gives dim and descending but no stable flat. L3344-3346 explicitly set stable=True.

Reply:
Would it be possible to account for this in the test cases rather than eliminating some combinations of inputs?